src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -40,14 +41,103 @@ public sealed class TensorFlowTransform : ITransformer, ICanSaveModel
     {
         public sealed class Arguments : TransformInputBase
         {
+            /// <summary>
+            /// Location of the TensorFlow model.
+            /// </summary>
             [Argument(ArgumentType.Required, HelpText = "TensorFlow model used by the transform. Please see https://www.tensorflow.org/mobile/prepare_models for more details.", SortOrder = 0)]
             public string Model;
 
+            /// <summary>
+            /// The names of the model inputs.
+            /// </summary>
             [Argument(ArgumentType.Multiple | ArgumentType.Required, HelpText = "The names of the model inputs", ShortName = "inputs", SortOrder = 1)]
             public string[] InputColumns;
 
+            /// <summary>
+            /// The names of the requested model outputs.
+            /// </summary>
             [Argument(ArgumentType.Multiple | ArgumentType.Required, HelpText = "The name of the outputs", ShortName = "outputs", SortOrder = 2)]
             public string[] OutputColumns;
+
+            /// <summary>
+            /// The name of the column used as label for training.
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Training labels.", ShortName = "label", SortOrder = 4)]
+            public string LabeLColumn = DefaultColumnNames.Label;
```

Comment:
>LabeLColumn [](start = 26, length = 11)

This must be identical to the name of the node in the graph corresponding to the label, correct?
If this is the case, then I don't think we should have the default value be "Label".

Reply:
Actually, I think it would make more sense to have two arguments: one for the name of the label column in the data view, and one for the name of the label node in the graph (unless it is possible to find it in the graph without user input?).

---
In reply to: [222725910](https://github.com/dotnet/machinelearning/pull/1063#discussion_r222725910) [](ancestors = 222725910)

Reply:
This is the same way the `InputColumns` and `OutputColumns` are treated. Do you think we should treat `Label` differently? What I can do is make it  required.

---
In reply to: [222726613](https://github.com/dotnet/machinelearning/pull/1063#discussion_r222726613) [](ancestors = 222726613,222725910)

Reply:
I don't think we should treat it differently, I think we should also make that change for input/output columns :-).


---
In reply to: [222754435](https://github.com/dotnet/machinelearning/pull/1063#discussion_r222754435) [](ancestors = 222754435,222726613,222725910)

Reply:
I think we already have #769 for tracking this. I can add work for label column as well.

---
In reply to: [222841987](https://github.com/dotnet/machinelearning/pull/1063#discussion_r222841987) [](ancestors = 222841987,222754435,222726613,222725910)